pages/linux/toolbox-init-container.md

```diff
@@ -0,0 +1,9 @@
+# toolbox init-container
+
+> Initialize a running `toolbox` container.
+> This command is not expected to be directly invoked by humans, and cannot be used on the host.
```

Comment:
While it sounds sci-fi, I am not happy with this wording.

```suggestion
> This command is should not be executed by the user, and cannot be run on the host.
```

Reply:
This looks better